docs/.vuepress/theme/components/Page.vue

```diff
@@ -47,7 +47,11 @@ export default {
 
       if (selectedTab.tab.computedId.startsWith('preview-tab')) {
         this.activatedExamples.push(exampleId);
-        parentContainer.className += ` ${changedClass}`; // Adding class by method compatible with IE
+
+        // Extra timeout as there is some problem related to not adding class properly (#9856).
+        setTimeout(() => {
```

Comment:
As you already notice, that line of code is disturbing 😄 That's why I would propose to remove the code that directly mutates the DOM to code that is managed by the Vue itself. I've prepared working PoC https://github.com/handsontable/handsontable/commit/070117d7117dbef3081468e33556a06ebc3f9114. WDYT?

Reply:
> As you already notice, that line of code is disturbing 😄 That's why I would propose to remove the code that directly mutates the DOM to code that is managed by the Vue itself. I've prepared working PoC [070117d](https://github.com/handsontable/handsontable/commit/070117d7117dbef3081468e33556a06ebc3f9114). WDYT?

Thanks for your changes. It looks very good not and the workaround has been removed. I can see that class is added on `div` containing the main component which looks better. I take your code, it fulfills the assumptions much better!